torch/onnx/symbolic_opset11.py

```diff
@@ -983,3 +985,28 @@ def repeat_interleave(g, self, repeats, dim=None, output_size=None):
     # along the dimension provided, some post-processing is required
     loop_out = g.op("Transpose", loop_out, perm_i=perm_i)
     return reshape(g, loop_out, g.op("Constant", value_t=torch.LongTensor(output_sizes)))
+
+def broadcast_tensors(g, self):
+    all_tensors = sym_help._unpack_list(self)
```

Comment:
Maybe could add a check here along the lines
if sym_help._is_packed_list(self)

Reply:
I agree. Do we expect the 'self' argument to be a packed_list always?

Reply:
Yes, we do.

Reply:
Can we add assert?

Reply:
Since we expect the 'self' argument to be a packed_list always, we don't need to add an extra assert here. Such check has been done in sym_help._unpack_list() method already.

Per my understanding, the usage of adding an assert here is to handle the case that self argument is not a packed_list. But this is not what we expected so we should fail in such case instead of creating a new list which contains only 1 element(self argument).

Reply:
So assert is exactly for when we expect a behavior.
If you expect it to be packed_list, you put an assert, right?
This is not "handle" the case if input is not packed_list. It is used simply to assert the expectation, and guard for any (future) bugs or even changes (possibly in the upstream).
It just makes debugging easier. In any case, if it's already done in sym_help._unpack_list(), it's okay.